apps/image-editor/src/js/command/setMosaic.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @author jkcs
+ * @fileoverview Set/Reset image(main) data for mosaic
+ */
+import commandFactory from '@/factory/command';
+import { commandNames } from '@/consts';
+
+const command = {
+  name: commandNames.SET_MOSAIC,
+
+  /**
+   * Set/Reset image src
+   * @param {Graphics} graphics - Graphics instance
+   * @param {string} imgUrl - Image Url
+   * @returns {Promise}
+   */
+  execute(graphics, imgUrl, type) {
+    const prevImage = graphics.getCanvasImage();
+    const prevImageWidth = prevImage ? prevImage.width : 0;
+    const prevImageHeight = prevImage ? prevImage.height : 0;
+    const objects = graphics
+      .removeAll(false)
+      .filter((objectItem) => objectItem.type !== 'cropzone');
```

Comment:
```suggestion
      .filter(({ type }) => type !== 'cropzone');
```
how about this?

Reply:
Blocking rules need to be added
// eslint-disable-next-line no-shadow

Reply:
You're right. I didn't see the `type` above. 😅